@xen-orchestra/lite/src/components/UsageBar.vue

```diff
@@ -87,16 +75,16 @@ const computedData = computed(() => {
 </style>
 
 <style>
+.progress-bar-component:nth-of-type(1) .progress-bar-fill,
+.progress-bar-component:nth-of-type(1) .circle {
+  background-color: var(--color-extra-blue-d60);
+}
 .progress-bar-component:nth-of-type(2) .progress-bar-fill,
 .progress-bar-component:nth-of-type(2) .circle {
-  background-color: var(--color-extra-blue-d60);
+  background-color: var(--color-extra-blue-d40);
```

Comment:
Why those changes?

Reply:
Since I removed the first `div`, the selectors based on an `nth` needed to be shifted by one.

Reply:
Got it :+1: